modules/common/src/test/scala/surge/internal/health/windows/actor/HealthSignalWindowActorSpec.scala

```diff
@@ -17,11 +17,11 @@ import surge.internal.health.windows.WindowSlider
 import surge.internal.health.{ HealthSignalBus, HealthSignalBusInternal }
 
 import scala.concurrent.duration._
-import scala.languageFeature.postfixOps
 class HealthSignalWindowActorSpec
     extends TestKit(ActorSystem("HealthSignalWindowActorSpec", ConfigFactory.load("artery-test-config")))
     with AnyWordSpecLike
     with Matchers
+    with BeforeAndAfterEach
```

Comment:
Doesn't look like this is used at the moment

Reply:
done